Grid UI tests migration P2.

denis-anisimov opened the pull request:
Fixes vaadin/framework8-issues#588

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/framework/8113)
<!-- Reviewable:end -->

Commit:
Grid UI tests migration P2.

Fixes vaadin/framework8-issues#588

elmot commented:
Reviewed 18 of 18 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[uitest/src/main/java/com/vaadin/tests/widgetset/client/grid/GridCellFocusOnResetSizeWidget.java, line 93 at r1](https://reviewable.io:443/reviews/vaadin/framework/8113#-K_TK1iRbyiIPM6f32ra:-K_TK1iRbyiIPM6f32rb:bpk1yhh) ([raw file](https://github.com/vaadin/framework/blob/c278ec83fc3894893f2a0b1dc8be54c41752c213/uitest/src/main/java/com/vaadin/tests/widgetset/client/grid/GridCellFocusOnResetSizeWidget.java#L93)):*
> ```Java
>     private class Col extends Grid.Column<String, String[]> {
>         public Col(String header) {
>             super(header);
> ```

Now we do not have this constructor. Perhaps the patch needs rebasing

---

*[uitest/src/main/java/com/vaadin/tests/widgetset/client/grid/GridDataChangeHandlerWidget.java, line 97 at r1](https://reviewable.io:443/reviews/vaadin/framework/8113#-K_TKQhNI_Rs80VzGai7:-K_TKQhNI_Rs80VzGai8:b9x2p2u) ([raw file](https://github.com/vaadin/framework/blob/c278ec83fc3894893f2a0b1dc8be54c41752c213/uitest/src/main/java/com/vaadin/tests/widgetset/client/grid/GridDataChangeHandlerWidget.java#L97)):*
> ```Java
>         grid.setDataSource(new RemoteDelayedDataSource(
>                 Arrays.asList("A", "B", "C", "D", "E")));
>         grid.addColumn(new Grid.Column<String, String>("letter") {
> ```

Rebase?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/8113)*
<!-- Sent from Reviewable.io -->

denis-anisimov commented:
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[uitest/src/main/java/com/vaadin/tests/widgetset/client/grid/GridCellFocusOnResetSizeWidget.java, line 93 at r1](https://reviewable.io:443/reviews/vaadin/framework/8113#-K_TK1iRbyiIPM6f32ra:-K_TjBJ7mKPEhEure-z7:b6d1xu5) ([raw file](https://github.com/vaadin/framework/blob/9003cfa8fddd5ecf4f4b8829dc4f51d44de58e96/uitest/src/main/java/com/vaadin/tests/widgetset/client/grid/GridCellFocusOnResetSizeWidget.java#L93)):*
<details><summary><i>Previously, elmot (Ilia Motornyi) wrote…</i></summary><blockquote>

Now we do not have this constructor. Perhaps the patch needs rebasing
</blockquote></details>

Rebased.
No changes.
This is client side.

---

*[uitest/src/main/java/com/vaadin/tests/widgetset/client/grid/GridDataChangeHandlerWidget.java, line 97 at r1](https://reviewable.io:443/reviews/vaadin/framework/8113#-K_TKQhNI_Rs80VzGai7:-K_TjG0Fzy4aWxqL8JVm:bncvegj) ([raw file](https://github.com/vaadin/framework/blob/9003cfa8fddd5ecf4f4b8829dc4f51d44de58e96/uitest/src/main/java/com/vaadin/tests/widgetset/client/grid/GridDataChangeHandlerWidget.java#L97)):*
<details><summary><i>Previously, elmot (Ilia Motornyi) wrote…</i></summary><blockquote>

Rebase?
</blockquote></details>

Rebased.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/8113)*
<!-- Sent from Reviewable.io -->

elmot commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/framework/8113#-:-K_TzIvuha-XJZQuJQGk:bnfp4nl)*
<!-- Sent from Reviewable.io -->
